projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -60,7 +60,7 @@ export class DotFormComponent {
         const { name, value } = event.detail;
         const process = fieldCustomProcess[tagName];
         if (tagName === 'DOT-BINARY-FILE' && value) {
-            this.uploadFile(event).then(tempFile => {
+            this.uploadFile(event).then((tempFile) => {
```

Comment:
type?

Reply:
this was worked by @hmoreras, no idea if there's a type since you can upload several files types...